packages/tidy/src/__tests__/index.test.ts

```diff
@@ -1,17 +1,74 @@
 import { resolve } from 'path'
 import { testRule } from '@sketch-hq/sketch-assistant-utils'
+import type { RuleConfig } from '@sketch-hq/sketch-assistant-types'
 
 import Assistant from '..'
+import { config } from '..'
 
-test('hello-world', async () => {
-  expect.assertions(2)
-
+const testCoreRuleWithConfig = async (fixture: string, ruleId: string, numViolations = 1) => {
```

Comment:
Smart 💡 👍

Reply:
A lot of ways to skin this cat! I guess that's a good thing, means it's pretty flexible.